sunkit_pyvista/tests/test_pyvista.py

```diff
@@ -96,8 +96,8 @@ def test_plot_coordinates(plotter):
                       distance=1*const.R_sun,
                       frame='heliocentricinertial')
     plotter.plot_coordinates(sphere)
-    assert plotter.plotter.mesh.n_cells == 1680
-    assert plotter.plotter.mesh.n_points == 842
+    assert plotter.plotter.mesh.n_cells == 1
+    assert plotter.plotter.mesh.n_points == 3
```

Comment:
I think that a spline is being plotted here instead of a sphere

Reply:
Thanks for catching this! Should be fixed now.